### DISCUSSION TITLE
Android build inclusion in dotnet-core build

### DISCUSSION THREAD

**Discussion by sripley52** [jedisct1/libsodium#1333] [Q&A]
I'm new to Android development so this may not make sense, but why aren't the Android builds included in the dotnet-core build (.github/workflows/dotnet-core)?

**Comment by jedisct1** [comment]
Because I don't know anything about .NET nor Android, so I'd rather let people who actually do contribute this.

Do .NET bindings (NSec, BlazorSodium, etc.) support Android?

**Reply by samuel-lucas6** [reply]
> Do .NET bindings (NSec, BlazorSodium, etc.) support Android?

Not out the box, and the same for iOS. You need to build libsodium yourself and modify the binding slightly, particularly for iOS. I don't think any of the people maintaining bindings have experience working with Android or iOS in .NET. It would be good to get this working. Related issues:

https://github.com/ektrah/nsec/issues/73
https://github.com/ektrah/libsodium-core/issues/80
https://github.com/samuel-lucas6/Geralt/issues/7
